frontend/src/tests/application-tests.js

```diff
@@ -23,9 +114,6 @@ export function applicationsTests({ apiGET, apiPOST }) {
         it.todo(
             "Even if a different utorid is submitted via survey.js data the active_user's utorid is used"
         );
-        it.todo(
-            "When submitting survey.js data cannot add a position_preference for a position not listed in the posting"
-        );
```

Comment:
Sorry, what was the reason we deleted this one?

Reply:
Hmm thought I pushed back in. Did it again.